Core/tooling/fix-bundles.js

```diff
@@ -70,9 +70,10 @@ const replace = (libSrc, regex, replacement) => {
   return libSrc.replace(regex, replacement);
 }
 
-let openpgpLib = fs.readFileSync('source/lib/openpgp.js').toString();
+let openpgpLib = fs.readFileSync('./node_modules/openpgp/dist/node/openpgp.js').toString();
 const openpgpLibNodeDev = openpgpLib; // dev node runs without any host, no modifications needed
 
+/*
```

Comment:
This will break the version of this code that builds for the iOS app (likely make it very slow).

The purpose is to offload intensive operations to Swift/ObC/C code that does it faster.

Can do in another PR.

Reply:
Since Swift based tests are passing - I assume it doesn't break. But it can slowdown.
But because library internals have significantly changed, this code was failing. So I've commented it out.
Also I've read somewhere (not sure now where exactly) that openpgp v5 automatically uses some native stuff if it is available.

Reply:
There is no browser where we run this code. There is `JavaScriptCore` which is a pure JS engine without any bells or whistles - no window, no node, no setTimeout or setInterval, no ... etc. We'll still need to supply our own crypto methods, unfortunately. It would be great if we didn't have to :-)

Reply:
> Since Swift based tests are passing - I assume it doesn't break. But it can slowdown.

They don't take this new code into account yet - still using previously built version, until we replace that manually, which we'll do in another PR. It will be an effort on its own.

Reply:
ok

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1379